tests/pulses/multi_channel_pulse_template_tests.py

```diff
@@ -319,8 +319,8 @@ def test_build_waveform_none(self):
         self.assertIsNone(wf)
 
     def test_deserialize(self):
-        sts = [DummyPulseTemplate(duration='t1', defined_channels={'A'}, parameter_names={'a', 'b'}),
-               DummyPulseTemplate(duration='t1', defined_channels={'B'}, parameter_names={'a', 'c'})]
+        sts = [DummyPulseTemplate(duration='t1', defined_channels={'A'}, parameter_names=['a', 'b']),
+               DummyPulseTemplate(duration='t1', defined_channels={'B'}, parameter_names=['a', 'c'])]
```

Comment:
Why this?

Reply:
I actually can't tell. Reverted in most current commit.